### PR TITLE
Fpad 984 take 2

### PR DESCRIPTION
**Pull request by tbrown224**

### CODE REVIEW

File: logging/log-events.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  FailedToReformatRecord = "Failed to Reformat Record",
+  FullSETBatchGenerated = "SET Batch Generation Succesful",
+  FailedSETBatchGenerated = "SET Batch Generation Failed",
+  PartialSETBatchGenerated = "SET Batch Generation Partially Successful",
+  JWSSignSuccess = "Successfully signed JWS",
+  JWEEncryptSuccess = "Successfully encrypted JWE",
+  PublicKeyRequested = "Public Key requested",
+  PublicKeyReturned = "Public Key returned successfully",
+  PublicKeyRequestFail = "Failed to get Public Key",
+}
```
Comment (aholwellGDS) [inline logging/log-events.ts:21]: I think the log events should probably remain in the specific repo that uses them. All apart from the top 3 at least as they're not really general.

**Review by aholwellGDS — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yaml [hunk 2]
```diff
@@ -18,7 +18,9 @@ jobs:
         with:
           node-version: 20
       - run: npm install
-      - run: npm run build
+      - run: npm ci
```
[the remark this answers is not in the document]
Comment (tbrown224) [reply .github/workflows/publish.yaml:21]: Done

File: package.json
[the remark this answers is not in the document]
Comment (tbrown224) [reply package.json:1]: Removed

File: logging/package.json
[the remark this answers is not in the document]
Comment (tbrown224) [reply logging/package.json:1]: Removed package.json from root folder and kept the package.json within /logging directory

File: jest.config.ts
[the remark this answers is not in the document]
Comment (tbrown224) [reply jest.config.ts:1]: Done

File: logging/logging.ts
[the remark this answers is not in the document]
Comment (tbrown224) [reply logging/logging.ts:1]: This is out of scope of this ticket and will therefor raise a separate ticket to address these changes

**Review by aholwellGDS — APPROVED** [review]

**Pull request merged**